webcalc/templates/about.html

```diff
@@ -1,9 +1,153 @@
+
+                    {% latexify 'PosBiScore(w=x_1 \dots x_n) \approx 1 + \sum_{i=2}^{n} P(w_{i-1} = x_{i-1}, w_i = x_i)' math_block=True %}
+
+                    where
+
+                    {% latexify 'P(w_{i-1} = y, w_i = x) = \frac{C(w_{i-1} = y, w_i = x)}{\displaystyle\sum_{z \in \Sigma}\sum_{v \in \Sigma} C(w_{i-1} = z, w_i = v)}' math_block=True %}
+
+                    where {% latexify 'w_i' math_inline=True %} refers to the {% latexify 'i^{\text{th}}' math_inline=True %} position in a word and {% latexify 'C(w_{i-1} = y, w_i = x)' math_inline=True %} is the number of times in the training data the sequence {% latexify 'yx' math_inline=True %} occurs at the {% latexify '(i-1)^{\text{th}}' math_inline=True %} and {% latexify 'i^{\text{th}}' math_inline=True %} positions of a word.
+                </p>
+                <p>
+                    Under this metric, the score assigned to a word is based on the sum of the probability of each contiguous pair of symbols occuring at their respective positions. Higher scores represent words with more probable phonotactics, but note that this scores cannot be interpreted as probabilities.
+                </p>
+            </li>
+            <li>
+                <p>
+                    <b>Token-weighted variants</b>: Assuming that the training data consists of a list of word types (e.g., a dictionary), the above metrics can be described as <i>type-weighted</i>: the frequency of individual word types has no bearing on the scores assigned by the metrics.
+                </p>
+                <p>
+                    The calculator also includes <i>token-weighted</i> variants of each of the above measures, where the phonotactic properties of frequent word types are weighted higher than those in less frequent word types. These are included under the column names <tt>uni_tok_prob, bi_tok_prob, pos_uni_tok, pos_bi_tok</tt>.
+                </p>
+                <p>
+                    These measures are computed by changing the count function {% latexify 'C' math_inline=True %} such that it is the number of occurrences of the configuration in question multiplied by the natural log of the count of the word containing each occurrence.
+                </p>
+                <p>
+                    For example, suppose we have a corpus containing two word types "kæt", which occurs 1000 times, and "tæk", which occurs 50 times. Under a token-weighted unigram model, {% latexify 'C(æ) = ln(1000) + ln(50) \approx 10.82' math_inline=True %}, while in a type-weighted unigram model {% latexify 'C(æ) = 1 + 1 = 2' math_inline=True %}. 
+                </p>
+                <p>
+                    The token-weighted positional ungiram and bigram scores correspond to the metrics presented in Vitevich and Luce (2004; though they use the base-10 logarithm rather than the natural logarithm).
```

Comment:
Not sure if this is right but I think the formatting of this sentence should be as follows:
... Vitevich and Luce (2004), though they use the base-10 logarithm rather than the natural logarithm.